index.html

```diff
@@ -54,6 +54,208 @@
+                </div>
+              </div>
+              <div id="works-section-4-block-apart-primary-text-2">
+                <p id="works-section-4-block-apart-primary-text-2-p"></p>
+              </div>
+              <div id="works-section-4-block-apart-tags">
+                <ul id="block-apart-tags-li">
+                  <li id="block-apart-tags-item-1" class="block-apart-tags"><a class="block-apart-tags-item">html</a></li>
+                  <li id="block-apart-tags-item-aux" class="block-apart-tags-aux"><a class="block-apart-tags-item-aux"></a></li>
+                  <li id="block-apart-tags-item-2" class="block-apart-tags"><a class="block-apart-tags-item">css</a></li>
+                  <li id="block-apart-tags-item-3" class="block-apart-tags"><a class="block-apart-tags-item">javaScript</a></li>
+                </ul>
+              </div>
+              <div id="works-section-4-block-apart-action"><button id="block-apart-action-button" type="button">See Project</button></div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <!-- about-myself -->
+      <section id="about-myself-base"></section>
+
+      <section id="about-myself">
+        <div id="about-myself-top">
+          <div id="about-myself-top-primary-text"><h2 class="about-myself-top-primary-text-1">About<br>Myself</h2></div>
+          <div id="about-myself-top-primary-text-2">
+            <p class="about-myself-top-primary-text-2">Hello ľm  a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          </div>
+          <div id="about-myself-top-social-media"><h5 class="social-media-p">LET'S CONNECT</h5></div>
+
+          <div id="about-myself-top-social-media-icons">
+            <ul id="social-media-icons-li-2">
+              <li id="social-media-icons-item-1">
+                <a href="https://github.com/alex1779/"><img class="social" src="images/github.svg" alt="github" width="auto" height="auto"></a>
+              </li>
+              <li id="social-media-icons-item-2">
+                <a href="https://www.linkedin.com/in/alejandro-maggioni-086678b5/"><img class="social" src="images/linkedin.svg" alt="linkedin" width="auto" height="auto"></a>
+              </li>
+              <li id="social-media-icons-item-3">
+                <a href="#"><img class="social" src="images/hand.svg" alt="hand" width="auto" height="auto"></a>
+              </li>
+              <li id="social-media-icons-item-4">
+                <a href="https://twitter.com/AlejandroMaggi7"><img class="social" src="images/twitter.svg" alt="twitter" width="auto" height="auto"></a>
+              </li>
+              <li id="social-media-icons-item-5">
+                <a href="#"><img class="social" src="images/m.svg" alt="m" width="auto" height="auto"></a>
+              </li>
+            </ul>
```

Comment:
- [ ] Great job! 👏 However, your `LET'S CONNECT` heading and your `social-media-icons` are overlapping. Please would you mind fixing that?
![Screenshot from 2022-11-29 22-45-42](https://user-images.githubusercontent.com/89228743/204665864-d9337fde-180b-45a8-b2c4-c74a7aca21b0.png)

Reply:
@ridwanediallo Thanks! 
What browser are you using to see the error?? I'm using Google Chrome and I can't see that error. Everything seems ok by my side.